### DISCUSSION TITLE
Conditional rendering in a Stack causes Hook error

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#866] [Q&A] [answered]
sandbox: https://codesandbox.io/s/conditional-rendering-in-a-stack-tms0j

Clicking the button conditionally renders an Alert, which then crashes the app due to this hook in Stack:

https://github.com/twilio-labs/paste/blob/main/packages/paste-core/layout/stack/src/index.tsx#L85-L91

**Comment by SiTaggart** [comment, marked as answer]
Yup, this is a known bug. For reference: https://github.com/twilio-labs/paste/issues/808 and https://github.com/twilio-labs/paste/discussions/838

Working on it in https://github.com/twilio-labs/paste/pull/825 which should be merged by Wednesday next week.
